### PR TITLE
Refactor/get fetch

### LINKED ISSUE #23: Post fetch data
As a user I would like to add data to the app for steps, hydration, and sleep quality.

### PR DESCRIPTION
**Pull request by cameronRomo**
## What is the change?
* Refactored `apiRequest.js` to be an object with only methods

## What does it fix? 
* Methods for 'GET' can now be invoked in the `scripts.js`

## Is this a fix or a feature?  
* Both this PR offers both a fix and a future

## Where should the reviewer start?
* `apiRequest.js`
  * lines 1 - 31
* `scripts.js`
  * lines 30 - 35

## How should this be tested?

### CODE REVIEW

File: src/api-request.js [hunk 1]
```diff
@@ -1,42 +1,31 @@
...
-  activityInfo: getActivityData()
-};
-function getUserData() {
-  return fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
+getUserData() {
+    return fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
     .then(response => response.json())
     .then(data => data.userData)
     .catch(error => console.log(error));
+},
```
Comment (okalimbet) [inline src/api-request.js:7]: That's an interesting way to organize this data inside the object literal! I learned something new!

File: src/scripts.js [hunk 2]
```diff
@@ -58,17 +42,6 @@ Promise.all([fetchedUserData, fetchedSleepData, fetchedActivityData, fetchedHydr
   })
   .catch(error => console.log(error))
 
-// Promise.all([apiRequest.userInfo, apiRequest.sleepInfo, apiRequest.activityInfo, apiRequest.hydrationInfo])
```
Comment (okalimbet) [inline src/scripts.js:61]: Thank you for cleaning up the script.js file

File: src/api-request.js [hunk 3]
```diff
@@ -1,42 +1,31 @@
 let apiRequest = {
-  userInfo: getUserData(),
-  sleepInfo: getSleepData(),
-  hydrationInfo: getHydrationData(),
-  activityInfo: getActivityData()
-};
-function getUserData() {
-  return fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
+getUserData() {
```
Comment (mdflynn) [inline src/api-request.js:2]: Oh I see. create the methods within the object literal instead of invoking them inside of it.

File: src/scripts.js [hunk 4]
```diff
@@ -27,28 +27,12 @@ let userHydrationInput = $('#user__hydration__input');
...
+const recievedActivityData = apiRequest.getActivityData();
+const recievedHydrationData = apiRequest.getHydrationData();
 
-const fetchedHydrationData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData')
-  .then(response => response.json())
-  .then(data => data.hydrationData)
-  .catch(error => console.log(error));
-
-
-Promise.all([fetchedUserData, fetchedSleepData, fetchedActivityData, fetchedHydrationData])
```
Comment (okalimbet) [inline src/scripts.js:51]: Glad we could make Promise.all work!

**Review by okalimbet — APPROVED** [review]

**Review by mdflynn — APPROVED** [review]
Nice work getting the data to work across files.

**Pull request merged**